docs/docker-image-details.md

```diff
@@ -168,6 +168,10 @@ Changelog
 You can use this kind of image to test the catalog of Apache Hive.
 
 Changelog
+
+- apache/gravitino-ci:hive-0.1.17
+  - Add support for JDBC SQL standard authorization
+- 
```

Comment:
Please add description of the `hive-site.xml` and `hiveserver2-site.xml` in the here.

Reply:
Ok, I added refer to other image version records.